qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -858,6 +877,23 @@ compare_images()
     rm -f ${rmt_export} ${loc_export}
 }
 
+compare_image_snapshots()
+{
+    local pool=$1
+    local image=$2
+
+    local rmt_export=${TEMPDIR}/${CLUSTER2}-${pool}-${image}.export
+    local loc_export=${TEMPDIR}/${CLUSTER1}-${pool}-${image}.export
+
+    for snap_name in $(rbd --cluster ${CLUSTER1} -p ${pool} snap list ${image}); do
+        rm -f ${rmt_export} ${loc_export}
+        rbd --cluster ${CLUSTER2} -p ${pool} export ${image}@${snap_name} ${rmt_export}
+        rbd --cluster ${CLUSTER1} -p ${pool} export ${image}@${snap_name} ${loc_export}
+        cmp ${rmt_export} ${loc_export}
```

Comment:
@dillaman You could do something like this:
```
rmt_md5sum=`rbd ... export image - | md5sum`
```
And just compare check sums. It looks like less code and probably a little faster.

Reply:
I was keeping it consistent w/ https://github.com/ceph/ceph/pull/21697/files/1d5b146cfa6c72f02860f15d1ec330ff97ce0848..de4587012d2d0d35f9d201550d58ffc2faeff27b#diff-1d6828c5e08d9d9d1f6dea4aa3430afaR876. Plus ```cmp``` will at least say the offset of the first delta byte.